JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -75,27 +75,39 @@ private val defaultSpacerSize = 16.dp
 
 @Composable
 fun PostContent(post: Post, modifier: Modifier = Modifier) {
-    ScrollableColumn(
+    LazyColumn(
         modifier = modifier.padding(horizontal = defaultSpacerSize)
     ) {
-        Spacer(Modifier.preferredHeight(defaultSpacerSize))
-        PostHeaderImage(post)
-        Text(text = post.title, style = MaterialTheme.typography.h4)
-        Spacer(Modifier.preferredHeight(8.dp))
+        item {
+            Spacer(Modifier.preferredHeight(defaultSpacerSize))
+            PostHeaderImage(post)
+        }
+        item {
+            Text(text = post.title, style = MaterialTheme.typography.h4)
+            Spacer(Modifier.preferredHeight(8.dp))
+        }
         post.subtitle?.let { subtitle ->
-            Providers(AmbientContentAlpha provides ContentAlpha.medium) {
-                Text(
-                    text = subtitle,
-                    style = MaterialTheme.typography.body2,
-                    lineHeight = 20.sp
-                )
+            item {
+                Providers(AmbientContentAlpha provides ContentAlpha.medium) {
+                    Text(
+                        text = subtitle,
+                        style = MaterialTheme.typography.body2,
+                        lineHeight = 20.sp
+                    )
+                }
+                Spacer(Modifier.preferredHeight(defaultSpacerSize))
             }
-            Spacer(Modifier.preferredHeight(defaultSpacerSize))
         }
-        PostMetadata(post.metadata)
-        Spacer(Modifier.preferredHeight(24.dp))
-        PostContents(post.paragraphs)
-        Spacer(Modifier.preferredHeight(48.dp))
+        item {
+            PostMetadata(post.metadata)
+            Spacer(Modifier.preferredHeight(24.dp))
+        }
+        items(post.paragraphs) {
+            Paragraph(paragraph = it)
```

Comment:
`Paragraph` wasn't there before

Reply:
PostContents used to contain a `forEach`, adding a paragraph composable for each paragraph. I replaced that inline, so I could use `items`. But I forgot to remove the now unused PostContents composable. I added a new commit with that.